CacheSim/CacheSimCommon.inl

```diff
@@ -0,0 +1,628 @@
+#pragma once
+
+///////////
+// This file contains common code for all platform
+// It must be included exactly ONCE for the entire library
+//////////
+
+#include "Precompiled.h"
+
+#include "CacheSim.h"
+#include "CacheSimInternals.h"
+#include "CacheSimData.h"
+#include "GenericHashTable.h"
+#include "Md5.h"
+
+extern "C"
+{
+#include "udis86/udis86.h"
+}
+
+#define ARRAY_SIZE(a) (sizeof(a)/sizeof(a[0]))
+
+
+
+namespace CacheSim
+{
+    enum 
```

Comment:
No worries - yeah, the Insomniac style is two spaces, and we try to follow ws inside parens etc per project. My style is `if (x)`, but some new code here uses `if ( x )`. If it helps perhaps just run astyle or clang-format or something, I don't want you to get lost in whitespace details.